diff --git a/node_modules/react-native-gesture-handler/ios/RNGestureHandlerButtonComponentView.mm b/node_modules/react-native-gesture-handler/ios/RNGestureHandlerButtonComponentView.mm
index af99949..cf8923e 100644
--- a/node_modules/react-native-gesture-handler/ios/RNGestureHandlerButtonComponentView.mm
+++ b/node_modules/react-native-gesture-handler/ios/RNGestureHandlerButtonComponentView.mm
@@ -47,7 +47,9 @@ using namespace facebook::react;
     const auto &newProps = *std::static_pointer_cast<const RNGestureHandlerButtonProps>(props);
 
     _buttonView.userEnabled = newProps.enabled;
+#if !TARGET_OS_TV
     _buttonView.exclusiveTouch = newProps.exclusive;
+#endif
 
     [super updateProps:props oldProps:oldProps];
 }
