diff --git a/node_modules/react-native-svg/RNSVG.podspec b/node_modules/react-native-svg/RNSVG.podspec
index afd13a1..6e4ef48 100644
--- a/node_modules/react-native-svg/RNSVG.podspec
+++ b/node_modules/react-native-svg/RNSVG.podspec
@@ -11,7 +11,7 @@ Pod::Spec.new do |s|
   s.license           = package['license']
   s.homepage          = package['homepage']
   s.authors           = 'Horcrux Chen'
-  s.platforms         = { :osx => "10.14", :ios => "9.0", :tvos => "9.2" }
+  s.platforms         = { :osx => "10.14", :ios => "12.4", :tvos => "12.4" }
   s.source            = { :git => 'https://github.com/react-native-community/react-native-svg.git', :tag => "v#{s.version}" }
 
   if fabric_enabled
@@ -23,7 +23,7 @@ Pod::Spec.new do |s|
       'HEADER_SEARCH_PATHS' => '"$(PODS_ROOT)/boost" "$(PODS_ROOT)/boost-for-react-native"  "$(PODS_ROOT)/RCT-Folly"',
       "CLANG_CXX_LANGUAGE_STANDARD" => "c++17",
     }
-    s.platforms       = { ios: '11.0', tvos: '11.0' }
+    s.platforms       = { ios: '12.4', tvos: '12.4' }
     s.compiler_flags  = folly_compiler_flags + ' -DRCT_NEW_ARCH_ENABLED'
     s.source_files    = 'apple/**/*.{h,m,mm,cpp}'
     s.ios.exclude_files = '**/*.macos.{h,m,mm,cpp}'
