diff --git a/node_modules/react-native-reanimated/RNReanimated.podspec b/node_modules/react-native-reanimated/RNReanimated.podspec
index a8d2d3d..1fbe191 100644
--- a/node_modules/react-native-reanimated/RNReanimated.podspec
+++ b/node_modules/react-native-reanimated/RNReanimated.podspec
@@ -86,7 +86,7 @@ Pod::Spec.new do |s|
   s.dependency 'React-Core'
   s.dependency 'React-CoreModules'
   s.dependency 'React-Core/DevSupport'
-  s.dependency 'React-RCTActionSheet'
+# s.dependency 'React-RCTActionSheet'
   s.dependency 'React-RCTNetwork'
   s.dependency 'React-RCTAnimation'
   s.dependency 'React-RCTLinking'
